rfcs/step_wait_promise.md

```diff
@@ -0,0 +1,86 @@
+# RFC 207: `step_wait_promise` method
+
+## Summary
+
+Add a `step_wait_promise(promise, description, timeout=default_timeout)` method to 
```

Comment:
would it be too confusing to support a promise in step_wait? `if (predicate instanceof Promise) { ... new behavior } else { ... old behavior ... }`?

Reply:
I'm afraid it is indeed too confusing